src/message/ping-message.h

```diff
@@ -2,24 +2,26 @@
 
 #pragma once
 #include <inttypes.h>
-#include <string.h>
 #include <stdlib.h>
+#include <string.h>
```

Comment:
can we split this file re-format into a separate commit? (no big deal, does not block)

Reply:
My fault, I did a clang-format and add -u without checking this files.

Reply:
Done